components/esp_websocket_client/examples/main/websocket_example.c

```diff
@@ -32,7 +32,7 @@
 
 #define NO_DATA_TIMEOUT_SEC 5
 
-static const char *TAG = "WEBSOCKET";
+static const char *TAG = "websocket";
```

Comment:
This need a fix in the test code, see the failure here: https://github.com/espressif/esp-protocols/actions/runs/4185017500/jobs/7256313908#step:6:292

Please update the `pytest`, as well